tests/syscall/fs/enc/enc.cpp

```diff
@@ -418,6 +415,34 @@ static void test_invalid_path(FILE_SYSTEM& fs)
     OE_TEST(fs.truncate(path, 0) == -1);
 }
 
+template <class FILE_SYSTEM>
+void test_common(FILE_SYSTEM& fs, const char* tmp_dir)
+{
+    cleanup(fs, tmp_dir);
+    test_create_file(fs, tmp_dir);
+    test_read_file(fs, tmp_dir);
+    //    test_pread_file(fs, tmp_dir);
```

Comment:
Should this line (and the following one) be removed?

Reply:
Yes must be. Thanks!